apps/api/v1/permissions.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import permissions
+
+
+class ResourcePermission(permissions.BasePermission):
+    message = 'Updating resource for like not allowed for anonymous users.'
+
+    def has_permission(self, request, view):
```

Comment:
Doesn't django-rest-framework offer anything else for this?

Reply:
We might be able to do this `@detail_route`